### DISCUSSION TITLE
Statistics from "non-canon" events

### DISCUSSION THREAD

**Discussion by duechayapolgmit** [Noxcrew/mcchampionship-api#13] [Ideas]
Hello,

Since events are not officially categorised between canon or non-canon any more, would it be possible if we have statistics from the events that are missing from Season 3 that were considered as non-canon events in the past by the community?

These events would include:
- MCC AF
- MCC P23
- MCC R2
- MCC TR
- MCC PARTY

These statistics would be appreciated as to have a central accurate information for those events, and to keep consistency as with the approach of not categorising events anymore.

**Comment by MagicA550** [comment]
Hi there!

I just confirmed that all 5 of these events are still available on our back-end, so this should be possible using the tech we worked on for MCC Live Predictions. I'll add it to my list and update here when complete (likely by the end of next week).

**Comment by duechayapolgmit** [comment]
That is grand, thank you so much!

Would it be possible as well for other Seasons 1 and 2 events? I doubt that there would be statistics (in a similar sense to what is shown right now for those seasons) for those events, which is why I did not mention it initially.

**Comment by MagicA550** [comment]
All good!

S1/2 events were not formally stored on MCC Live because the history feature was not made on initial release, hence why they have limited data. Data for those events was actually compiled manually and it was deemed too laborious to get individual game scores too. If there is sufficient demand, maybe we could crowdsource in the future, but for now the answer is sadly not.

As for the non-canons in S1/2, we can look into that but that would probably be more manual data limited to event-wide stats.

**Comment by duechayapolgmit** [comment]
That is completely understandable. Thank you so much for your responses!

**Comment by MagicA550** [comment]
Hey there! I'm glad to say we've managed to unhide the events you requested in your OP. We'll look into adding further events later down the line. Hope this helps!
